packages/react/src/slider/root/SliderRoot.tsx

```diff
@@ -10,15 +10,16 @@ import { sliderStyleHookMapping } from './styleHooks';
 import { useSliderRoot } from './useSliderRoot';
 import { SliderRootContext } from './SliderRootContext';
 import { useFieldRootContext } from '../../field/root/FieldRootContext';
+import { fixedForwardRef } from '../utils';
 
 /**
  * Groups all parts of the slider.
  * Renders a `<div>` element.
  *
  * Documentation: [Base UI Slider](https://base-ui.com/react/components/slider)
  */
-const SliderRoot = React.forwardRef(function SliderRoot(
-  props: SliderRoot.Props,
+const SliderRoot = fixedForwardRef(function SliderRoot<TValue extends number | number[]>(
```

Comment:
ReactForwardRef does not work well with generics.
So we need to create a wrapper function to apply the generic.

Reply:
@seloner I think we can remove generic SliderRoot component by removing generic in Props interfacce.

Check https://github.com/mui/base-ui/pull/1241/files#r1899037802, please